src/Stratis.Bitcoin.Features.Wallet/Wallet.cs

```diff
@@ -798,6 +798,12 @@ public class TransactionData
         [JsonProperty(PropertyName = "hex", NullValueHandling = NullValueHandling.Ignore)]
         public string Hex { get; set; }
 
+        /// <summary>
+        /// Propagation state of this transaction.
+        /// </summary>
+        [JsonProperty(PropertyName = "isPropagated", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
You don't need `NullValueHandling = NullValueHandling.Ignore` here as bool is never gonna be null, but false by default.
If you want it not to show in the json, use `bool?` with `NullValueHandling = NullValueHandling.Ignore` so if you don't set it, it's not serialized.